README.md

```diff
@@ -1 +1,26 @@
-# UIDependent
\ No newline at end of file
+# UIDependent
+
+This repository contains lib with UI sizes converters for iOS development
```

Comment:
Change to `This repository contains a library with UI sizes converters for iOS development"